apps/web/app/(app)/environments/[environmentId]/(actionsAndAttributes)/actions/components/ActionClassesTable.tsx

```diff
@@ -38,16 +43,20 @@ export default function ActionClassesTable({
 
   return (
     <>
-      <div className="mb-6 text-right">
-        <Button
-          variant="darkCTA"
-          onClick={() => {
-            setAddActionModalOpen(true);
-          }}>
-          <CursorArrowRaysIcon className="mr-2 h-5 w-5 text-white" />
-          Add Action
-        </Button>
-      </div>
+      <AccessWrapper loading={loading} error={error}>
```

Comment:
As the `AccessWrapper` is now not used for Access Management we should find another name for it. Maybe `LoadingWrapper`?